superset/examples/__init__.py

```diff
@@ -14,20 +14,20 @@
 # KIND, either express or implied.  See the License for the
 # specific language governing permissions and limitations
 # under the License.
-from .bart_lines import load_bart_lines  # noqa
-from .birth_names import load_birth_names  # noqa
-from .country_map import load_country_map_data  # noqa
-from .css_templates import load_css_templates  # noqa
-from .deck import load_deck_dash  # noqa
-from .energy import load_energy  # noqa
-from .flights import load_flights  # noqa
-from .long_lat import load_long_lat_data  # noqa
-from .misc_dashboard import load_misc_dashboard  # noqa
-from .multi_line import load_multi_line  # noqa
-from .multiformat_time_series import load_multiformat_time_series  # noqa
-from .paris import load_paris_iris_geojson  # noqa
-from .random_time_series import load_random_time_series_data  # noqa
-from .sf_population_polygons import load_sf_population_polygons  # noqa
-from .tabbed_dashboard import load_tabbed_dashboard  # noqa
-from .unicode_test_data import load_unicode_test_data  # noqa
-from .world_bank import load_world_bank_health_n_pop  # noqa
+from .bart_lines import load_bart_lines
+from .birth_names import load_birth_names
+from .country_map import load_country_map_data
+from .css_templates import load_css_templates
+from .deck import load_deck_dash
+from .energy import load_energy
+from .flights import load_flights
+from .long_lat import load_long_lat_data
+from .misc_dashboard import load_misc_dashboard
+from .multi_line import load_multi_line
+from .multiformat_time_series import load_multiformat_time_series
+from .paris import load_paris_iris_geojson
+from .random_time_series import load_random_time_series_data
+from .sf_population_polygons import load_sf_population_polygons
+from .tabbed_dashboard import load_tabbed_dashboard
+from .unicode_test_data import load_unicode_test_data
+from .world_bank import load_world_bank_health_n_pop
```

Comment:
Out of scope and I might be missing something obvious, but why are we importing relative `.bart_lines` instead of `superset.examples.bart_lines` etc? I assume superset will always be installed, and hence should be absolute-importable.

Reply:
Agree. I think we should always use absolute imports for consistency and readability.